footer.php

```diff
@@ -25,7 +25,7 @@
 			<a href="<?php echo esc_url( __( 'https://wordpress.org/', 'twentynineteen' ) ); ?>" class="imprint">
 				<?php
 				/* translators: %s: WordPress. */
-				printf( esc_html__( 'Proudly powered by %s.', 'twentynineteen' ), 'WordPress' );
+				printf( __( 'Proudly powered by %s.', 'twentynineteen' ), 'WordPress' );
```

Comment:
We could just change the string to `_e( 'Proudly powered by WordPress.', 'twentynineteen' )`
```suggestion
				_e( 'Proudly powered by WordPress.', 'twentynineteen' );
```

Reply:
There is precedent to this: https://github.com/WordPress/twentyseventeen/blob/master/components/footer/site-info.php#L13. I suppose to avoid "WordPress" from being changed / translated.